common/model/tests/test_cpptransaction.cc

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include <string.h>
```

Comment:
Either write tests (which you should) or remove this file.
Also, C++ tests use googletest